lib/utils/appList.js

```diff
@@ -32,7 +32,7 @@ const MAPPINGS = {
 };
 
 function extaractDeveloperId (link) {
```

Comment:
while you are at it can you 
```suggestion
function extractDeveloperId (link) {
```

Reply:
(and update the calling sites)

Reply:
Sorry, I didn't get what you mean.

Reply:
there's a typo in the function name `extAract` instead of `extract`